db/migrate/20151106120125_add_completed_to_application.rb

```diff
@@ -0,0 +1,22 @@
+class AddCompletedToApplication < ActiveRecord::Migration
+  def up
+    add_column :applications, :completed_at, :datetime
+    add_reference :applications, :completed_by, references: :users
+    migrate_sql = <<-SQL.gsub(/^\s+\|/, '')
+      |UPDATE applications
+      |SET
+      |  completed_at = "applications"."updated_at",
+      |  completed_by_id = "applications"."user_id"
+      |FROM "applications"
+      |WHERE application_outcome IS NOT NULL AND
+      |  status IS NOT NULL AND
+      |  application_type IS NOT NULL;
```

Comment:
Yeah, I think this is sufficient. Maybe one or two will slip in which shouldn't be there, but that's fine :)